src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -276,7 +276,7 @@ private void createConsumer(RoutingContext routingContext) {
         this.httpBridgeContext.setOpenApiOperation(HttpOpenApiOperations.CREATE_CONSUMER);
 
         // check for an empty body
-        JsonObject body = routingContext.getBody() != null && routingContext.getBody().length() != 0 ? routingContext.getBodyAsJson() : new JsonObject();
+        JsonObject body = routingContext.body().buffer() != null && routingContext.body().buffer().length() != 0 ? routingContext.getBodyAsJson() : new JsonObject();
```

Comment:
So has the new vertx version changed the inplementation of RoutingContext?

Reply:
Yeah they have deprecated the `getBody` and other methods, everything seems to be under a `RequestBody` object returned by `body` method.